records_mover/db/driver.py

```diff
@@ -83,7 +83,7 @@ def set_grant_permissions_for_users(self, schema_name: str, table: str,
     def supports_time_type(self) -> bool:
         return True
 
-    def type_for_date_plus_time(self, has_tz: bool=False) -> sqlalchemy.sql.sqltypes.DateTime:
+    def type_for_date_plus_time(self, has_tz: bool = False) -> sqlalchemy.sql.sqltypes.DateTime:
```

Comment:
From what I've seen so far the original codebase more commonly uses the first option, with no extra space (my personal preference as well), but not a big deal.

Reply:
Agreed stylistically but triggering flake8 so 🤷